tsdb/shard.go

```diff
@@ -452,6 +452,10 @@ func (s *Shard) validateSeriesAndFields(points []models.Point) ([]*FieldCreate,
 		key := string(p.Key())
 		ss := s.index.Series(key)
 		if ss == nil {
+			if len(s.index.series)-1 > s.options.Config.MaxSeriesPerDatabase {
+				return nil, fmt.Errorf("would exceed series limit: %s", key)
```

Comment:
Maybe "max series per database exceeded" which similar to https://github.com/influxdata/influxdb/blob/master/tsdb/engine/tsm1/cache.go#L18